toolkit/tools/imagegen/configuration/testdata/test_configuration.json

```diff
@@ -83,7 +89,10 @@
                     "ID": "MyRootfsB",
                     "Start": 1033,
                     "End": 2048,
-                    "FsType": "ext4"
+                    "FsType": "ext4",
+                    "Flags": [
+                        "dmroot"
```

Comment:
I didn't go through the remaining 4 PRs yet, so it might be done later: we should also extend our documentation to mention this flag.

Reply:
Hmm, was going to do docs as a final item, but it might be best to include the flag part in this commit... I'll add it.